packages/api/db/migration/20221118081755_alter_irrigation_task_table.js

```diff
@@ -45,9 +45,9 @@ export const down = async function (knex) {
 
   for (const row of task_rows) {
     await knex.raw(
-      `UPDATE 'irrigation_task' task SET hours = ${
+      `UPDATE "task" t SET duration = ${
         row.duration / 60
```

Comment:
@wasswarichard 

Is the previous state of task.duration = irrigation_task.duration/60 or is the previous state NULL? 
![image](https://user-images.githubusercontent.com/30075973/203592523-104f5b7e-c8cf-47eb-a392-9ffb9db44160.png)